include/dxc/HLSL/DxilShaderModel.h

```diff
@@ -86,7 +86,7 @@ class ShaderModel {
               unsigned m_NumInputRegs, unsigned m_NumOutputRegs,
               bool m_bUAVs, bool m_bTypedUavs, unsigned m_UAVRegsLim);
 
-  static const unsigned kNumShaderModels = 34;
+  enum { kNumShaderModels = 34 };
```

Comment:
I just checked in Debug and Release configurations, and it's gone in both cases. The name will be evaluated as an ICE everywhere, so unless someone takes the address or something crazy it should be pretty safe to assume this doesn't make it into the final image. I'd prefer to keep the simpler static const unless we find it's hurting something.

I you can push without this one change, I can merge this and then maybe do something about those make_unique's that we introduced elsewhere.

Thanks!